evaluation/benchmarks/poets/1_1.sh

```diff
@@ -2,4 +2,20 @@
 # tag: count_words
 
 IN=${IN:-$PASH_TOP/evaluation/benchmarks/poets/input/pg/}
-ls ${IN} | sed "s;^;$IN;" | xargs cat | tr -sc '[A-Z][a-z]' '[\012*]' | sort | uniq -c
+OUT=${OUT:-$PASH_TOP/evaluation/benchmarks/poets/output/1_1/}
+ENTRIES=${ENTRIES:-1000}
+mkdir -p "$OUT"
+#ls ${IN} | sed "s;^;$IN;" | xargs cat | tr -sc '[A-Z][a-z]' '[\012*]' | sort | uniq -c
+
+for input in $(ls ${IN} | head -n ${ENTRIES} | sed "s;^;$IN;")
+do
+    cat "$input" | tr -sc '[A-Z][a-z]' '[\012*]' | sort | uniq -c > "${OUT}/$(basename ${input})"
```

Comment:
Actually, there might be issues with the substitution `$(basename ...)`. I think that PaSh will reject compiling this.

@dkarnikis  Could we modify this to the following:
```sh
for input in $(ls ${IN} | head -n ${ENTRIES})
do
    cat "$IN/$input" | tr -sc '[A-Z][a-z]' '[\012*]' | sort | uniq -c > "${OUT}/${input}.out"
done
```